spectacles/validators.py

```diff
@@ -193,11 +193,21 @@ def validate(self, batch: bool = False) -> List[SqlError]:
         loop.run_until_complete(asyncio.sleep(0.250))
         loop.close()
 
-        running_task_ids, errors = self._get_query_results(query_task_ids)
-        while running_task_ids:
```

Comment:
I wonder if it makes it would be simpler to put this `while` in a `for` loop that iterates over the list in batches.

You could use something like [this](https://stackoverflow.com/a/312464/3524951). Thoughts?

Reply:
I think it would be less efficient. You'd have to wait for a full chunk to finish before kicking off the next one, wouldn't you? This way, we are always querying the max amount. (Let me know if I misunderstood the recommendation.)